README.md

```diff
@@ -0,0 +1,48 @@
+## What and Why?
+
+This is the Modern Tribe Extension Template. It allows you to easily extend our plugins: The Events Calendar, Event Tickets, and their associated add-on plugins.
+
+The primary benefits of using this template is that it's gets you up-and-running quicker than building a custom plugin yourself, and it's best to extend one plugin via another plugin instead of adding custom code to your child theme's `functions.php` file.
+
+We ensure your new extension does not run if its required base plugin is not activated, which helps avoid fatal errors and means fewer `class_exists()` checks are applicable.
+
+Extensions have been available since The Events Calendar version 4.3.3 (November 16, 2016). If you try to use this template to work with versions before this date, your extension will not run at all.
+
+## How?
+
+If you want to suggest an edit to this _template_, submit a pull request to this repo.
+
+#### If you want to make your own new extension:
+
+1. Create your own new repository, prefixing its name with the required `tribe-ext-` prefix (or else your extension will not run at all, even if activated).
+1. Download the `master` branch of _this template_ as a .zip and unzip it.
+1. Because you cannot make a new branch until you have an initial commit to your own new repo's `master` branch, copy this template's `license.txt` file and commit it to your repo's `master` branch.
+1. Create a new branch called `initial` or `v1` or whatever is appropriate and check out this branch.
+1. Place the remainder of this template's files in your repo's new branch.
+1. Find and replace these strings throughout all file names and file contents:
+    1. Extension plugin's slug: `tribe-ext-extension-template`
+    1. Extension plugin's slug (yes, again): `match-the-plugin-directory-name`
+    1. `[Base Plugin Name] Extension: [Extension Name]`:
+        1. `[Base Plugin Name]` should be something like `The Events Calendar`
+        1. `[Extension Name]` should be the name of your extension, such as `Remove Export Links`
+    1. `[Extension Description]` with your extension plugin's short description.
+    1. PHP namespace: `Tribe\Extensions\Example`
+    1. Extension class (must be unique among all extensions): `Tribe__Extension__Example`
```

Comment:
If we're encouraging PHP namespaces I'm not sure we also need double underscore pseudo-namespacing for the class name. Is this a limitation of the existing extension framework, or can we further simplify?